lib/iris/cube.py

```diff
@@ -1627,9 +1627,17 @@ def dtype(self):
     @property
     def fill_value(self):
         """
-        The fill value for the masked data array of this
-        :class:`~iris.cube.Cube`. If ``None``, the default NumPy fill value for
-        the dtype of the masked data array will be used.
+        A value suitable for filling in missing (i.e. masked) datapoints of
+        this :class:`~iris.cube.Cube` s data.
+
+        This can also be ``None``, meaning that no suitable fill value is
+        known.
+
+        .. Note::
+
+            Ideally, this value will not occur anywhere in `cube.data`. Thus,
+            many operations which change cube data will also set `fill_value`
```

Comment:
This is not the general which / that argument, surely, because this "which" is not a pronoun ?